hack/deploy-logging.sh

```diff
@@ -122,15 +125,17 @@ EXTERNAL_REGISTRY=${EXTERNAL_REGISTRY:-registry.svc.ci.openshift.org}
 EXT_REG_IMAGE_NS=${EXT_REG_IMAGE_NS:-origin}
 USE_OLM=${USE_OLM:-false}
 
-if [ -n "${OPENSHIFT_BUILD_NAMESPACE:-}" -a -n "${IMAGE_FORMAT:-}" ] ; then
-    USE_CUSTOM_IMAGES=${USE_CUSTOM_IMAGES:-true}
-elif [ "${USE_IMAGE_STREAM:-false}" = true ] ; then
-    USE_CUSTOM_IMAGES=${USE_CUSTOM_IMAGES:-true}
-elif [ "${USE_CLO_LATEST_IMAGE:-false}" = true -o "${USE_EO_LATEST_IMAGE:-false}" = true ] ; then
-    USE_CUSTOM_IMAGES=${USE_CUSTOM_IMAGES:-true}
-else
-    # default to false
-    USE_CUSTOM_IMAGES=${USE_CUSTOM_IMAGES:-false}
+if [ -z "${USE_CUSTOM_IMAGES:-}" ] ; then
+    if [ -n "${OPENSHIFT_BUILD_NAMESPACE:-}" -a -n "${IMAGE_FORMAT:-}" ] ; then
+        USE_CUSTOM_IMAGES=${USE_CUSTOM_IMAGES:-true}
```

Comment:
It does not matter, but we can have just `USE_CUSTOM_IMAGES=true` since USE_CUSTOM_IMAGES is guaranteed empty?

Reply:
not sure what you mean

Reply:
this is basically the same logic as we had before, but now we short circuit it if `USE_CUSTOM_IMAGES` is already set.

Reply:
still not sure what is the suggested change to this code

Reply:
Sorry for my slow response...  I just thought since USE_CUSTOM_IMAGES is always empty/unset when it comes to the assignment line, we could just set to true (without '${USE_CUSTOM_IMAGES:-' and '}'), which looks simpler.  But as it does not change the behaviour, it is not an issue at all.
```
if [ -z "${USE_CUSTOM_IMAGES:-}" ] ; then
    if [ -n "${OPENSHIFT_BUILD_NAMESPACE:-}" -a -n "${IMAGE_FORMAT:-}" ] ; then
        USE_CUSTOM_IMAGES=true
        ^^^^^^^^^^^^^^^^^^^^^^
```